lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -1852,7 +1852,7 @@ const spaceLimited = (
 						// So it should always end up being smaller
 						const headerSize = group.filteredChildren ? 2 : 1;
 						const limited = spaceLimited(
-							/** @type {Children<T>} */ (group.children),
```

Comment:
This is an error with TS 5.4, TS 5.5 introduced a bug that accidentally allows this call. At some point, that bug will get fixed (https://github.com/microsoft/TypeScript/pull/61013) so it would be good to fix this code before that happens :)